### PR TITLE
Various updates to text and content

### PR DESCRIPTION
**Pull request by richardmuscat**
- Updates to long and short descriptions
- Updates to AAG cards
- Various proofreading bits
- Removes screenshots

cc @jessefriedman

### CODE REVIEW

**Comment by richardmuscat** [comment]
Fixes: https://github.com/Automattic/jetpack/issues/4472
Fixes: https://github.com/Automattic/jetpack/issues/4615
Fixes: https://github.com/Automattic/jetpack/issues/4474

File: _inc/client/at-a-glance/backups.jsx [hunk 1]
```diff
@@ -50,8 +50,8 @@ const DashBackups = React.createClass( {
 
 						<p className="jp-dash-item__description">
 							{ vpData.message }
-							<br/>
-							{ __( '{{a}}View backup details{{/a}}', {
+
+							{ __( ' {{a}}View backup details{{/a}}.', {
```
Comment (oskosk) [inline _inc/client/at-a-glance/backups.jsx]: Just wondering if this space before the curly brackets `' {{a` is intentional. If it's for spacing purposes, it may be replaced by a `&nbsp;` before the JS code block

File: _inc/client/components/jetpack-connect/index.jsx [hunk 2]
```diff
@@ -18,18 +18,18 @@ const JetpackConnect = React.createClass( {
...
 				<Card className="jp-jetpack-connect__cta">
 					<p className="jp-jetpack-connect__description">
-						{ __( 'Please connect to or create a WordPress.com account to enable Jetpack, including powerful security, traffic, and customization services.' ) }
+						{ __( 'Please connect to or create a WordPress.com account to start using Jetpack. This will enable powerful security, traffic, and customization services.' ) }
 					</p>
 					<ConnectButton />
 					<p>
 						<a href="https://wordpress.com/start/jetpack/" className="jp-jetpack-connect__link">
-							{ __( 'No WordPress.com account? Create one for free.' ) }
+							{ __( 'No account? Create one for free...' ) }
```
Comment (oskosk) [inline _inc/client/components/jetpack-connect/index.jsx]: Apparently, the code is using the `…` character in general, more than the three dots. Although there are a few cases where the three dots are still present :)

File: _inc/client/components/jetpack-connect/index.jsx [hunk 3]
```diff
@@ -214,6 +214,11 @@ const JetpackConnect = React.createClass( {
 						) }
 					</p>
 					<ConnectButton />
+					<p>
+						<a href="https://wordpress.com/start/jetpack/" className="jp-jetpack-connect__link">
+							{ __( 'No account? Create one for free...' ) }
```
Comment (oskosk) [inline _inc/client/components/jetpack-connect/index.jsx]: Same thing here about `…` vs `...`

File: modules/module-info.php [hunk 4]
```diff
@@ -135,16 +107,9 @@ function publicize_load_more_link() {
...
-			<img class="jp-info-img" src="<?php echo plugins_url( basename( dirname( dirname( __FILE__ ) ) ) . '/images/screenshots/publicize.jpg' ) ?>" alt="<?php esc_attr_e( 'Publicize', 'jetpack' ) ?>" width="328" height="123" />
-		</a>
-	</div>
-
 	<?php esc_html_e(
-		'Publicize allows you to connect your site to social networks like Facebook, Twitter, Tumblr, Google+, Path,
-		and LinkedIn and automatically share new posts with your friends and followers. You can add connections for
-		yourself or for all users on your site.'
+		'Automatically share and promote newly published posts to Facebook, Twitter, Tumblr, Google+, Path,
+		and LinkedIn. You can add connections for	yourself or for all users on your site.'
```
Comment (oskosk) [inline modules/module-info.php]: There's a tab here between `for` and `yourself`

File: modules/module-info.php [hunk 5]
```diff
@@ -272,14 +210,9 @@ function jetpack_subscriptions_load_more_link() {
...
-	<div class="jp-info-img">
-		<img class="jp-info-img" src="<?php echo plugins_url( basename( dirname( dirname( __FILE__ ) ) ) . '/images/screenshots/subscriptions.jpg' ) ?>" alt="<?php esc_attr_e( 'Subsriptions Screenshot', 'jetpack' ) ?>" width="300" height="150" />
-	</div>
-
 	<?php esc_html_e(
-		'Allow any visitor to subscribe to your site through a widget in your sidebar. Each time you publish new content,
-		Jetpack will email a notification to all of your subscribers. When leaving comments, your visitors can also
-		subscribe to a post’s comments to keep up with the conversation.'
+		'A widget in your sidebar allows visitors to subscribe to your site so that they receive an email
+		each time you publish new content. Your visitors can also	subscribe to a post’s comments to keep up with the conversation.'
```
Comment (oskosk) [inline modules/module-info.php]: There's another tab here between `also` and `subscribe`

Comment (oskosk) [inline modules/module-info.php]: Ah also, `subscribe to post’s comments` is using the `’` character instead of `\'` as in other places.

File: modules/module-info.php [hunk 6]
```diff
@@ -348,15 +277,9 @@ function jetpack_contact_form_learn_more_button() {
...
-	</div>
-
 	<?php esc_html_e(
-		'Offer your readers the ability to get in touch without publishing your personal email address.
-		You can have multiple forms on your site and customize each one to fit specific needs.
-		When a user submits your contact form, their feedback will be emailed to you and added to
-		your feedback management area. If you have Akismet active on your site, submissions will be
+		'Create simple contact forms without any coding. You can have multiple forms and when
+		a user submits it, their feedback will be emailed directly to you. If Akismet is active, submissions will be
 		automatically filtered for spam. '
```
Comment (oskosk) [inline modules/module-info.php]: There's a space here after "spam. "

File: modules/module-info.php [hunk 7]
```diff
@@ -414,13 +328,9 @@ function jetpack_custom_css_more_button() {
...
 function jetpack_custom_css_more_info() { ?>
-	<div class="jp-info-img">
-		<img class="jp-info-img" src="<?php echo plugins_url( basename( dirname( dirname( __FILE__ ) ) ) . '/images/screenshots/custom-css.jpg' ) ?>" alt="<?php esc_attr_e( 'Custom CSS', 'jetpack' ) ?>" width="300" height="150" />
-	</div>
 	<?php esc_html_e(
-		"The Custom CSS editor lets you add to or replace your theme's CSS, while supplying syntax coloring,
-		auto-indentation, and immediate feedback on the validity of the CSS you're writing.
-		You can also add mobile styles, and we support the CSS preprocessors LESS and Sass."
+		"Add to or replace your theme's CSS including mobile styles, LESS, and SaSS.
+		Includes syntax coloring,	auto-indentation, and immediate CSS validation."
```
Comment (oskosk) [inline modules/module-info.php]: Another tab here before `auto-indentation`

File: modules/module-info.php [hunk 8]
```diff
@@ -755,9 +616,8 @@ function jetpack_sitemaps_more_link() {
 
 function jetpack_xml_sitemap_more_info() {
 	esc_html_e(
-		'Search engines like Google and Bing use sitemaps to crawl and understand your site making
-		it more likely for your content to show up on relevant searches. This feature creates two
-		sitemap files that list the URLs of posts and pages in your site with important information about each one.'
+		'Automatically create two	sitemap files that list the URLs of posts and pages in your site.
```
Comment (oskosk) [inline modules/module-info.php]: There's a tab character before `sitemap` here

File: modules/protect.php [hunk 9]
```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * Module Name: Protect
- * Module Description: Prevent and block malicious site attacks.
+ * Module Description: Prevent and block malicious login attempts.
```
Comment (oskosk) [inline modules/protect.php]: I like this replacement. Former text sounded a bit harsh!

**Comment by jessefriedman** [comment]
@richardmuscat did a great job updating the content. I found a few errors, and issues and committed fixes for them. As far as content goes (not code) this has a 👍 from me.

File: _inc/client/at-a-glance/stats.jsx [hunk 10]
```diff
@@ -304,10 +304,10 @@ const DashStatsBottom = React.createClass( {
 			</div>
 			<div className="jp-at-a-glance__stats-cta">
 				<div className="jp-at-a-glance__stats-cta-description">
-					<p>{ __( 'Need to see more stats, likes, followers, subscribers, and more?' ) }</p>
+					<p></p>
```
Comment (oskosk) [inline _inc/client/at-a-glance/stats.jsx]: empty <p> here

**Comment by jeffgolenski** [comment]
@richardmuscat  @jessefriedman 

Glad you guys are going through and tidying up the language a bit.

However, couple items to discuss with you! :shakesfist:

You've removed the helper text from the CTA from below the stats area, which was designed to text the users expectations. I think the change you've made will only prompt user surprise and frustration.

Let me explain:

**Before:**
screen shot 2016-08-11 at 5 54 24 pm

- Description tells the user what they can see
- CTA tells the user it will take them away from their dashboard (we communicate to user what's going to happen) 
screen shot 2016-08-11 at 5 57 04 pm

**After**


- "More" is ambiguous. Doesn't give the user much information.
- Takes them away from their dashboard. Not expected behavior & inconsistent with other links. The link directly below (manage security) tells the user its taking them to WPCOM, and this one doesn't. We should label all wpcom links as such as not to surprise users (and definitely maintain consistency throughout). Dragging folks to calypso from wp-admin is stressful enough to those who aren't accustomed to the wordpress.com interface, nevermind when we don't tell them we're doing it.

TL;DR  I think simplifying this area is going to have some negative repercussions. We should consider changing it back. 

cc @MichaelArestad

**Comment by MichaelArestad** [comment]
@jeffgolenski I don't remember why it was changed, but I'm wondering if it was changed because we wanted to use the Settings component without modifying it too much or something along those lines.

**Comment by MichaelArestad** [comment]
CC @dereksmart

**Comment by jeffgolenski** [comment]
> I don't remember why it was changed, but I'm wondering if it was changed because we wanted to use the Settings component without modifying it too much or something along those lines.

Not sure what you mean by that and how it relates to removing what was already built?  How is the text / button related to the settings component?

**Comment by MichaelArestad** [comment]
> Not sure what you mean by that and how it relates to removing what was already built? How is the text / button related to the settings component?

@jeffgolenski My suspicion is that what was previously built was a recreation of the component versus using the component and was becoming unwieldy (it's a complex monster). That said, I'm not the one to ask.

**Comment by akirk** [comment]
Please ping us when making big text changes like these. That cost us a lot of money because of large text re-translation. In such a case it might make sense to keep the old translations around so that we don't have to pay for re-translation.

**Comment by akirk** [comment]
(also the texts maintain the inappropriate whitespace in translations `\n\t\t` which make them hard to translate as well)
